package.json

```diff
@@ -46,7 +46,7 @@
   "dependencies": {
     "eslint-scope": "^4.0.0",
     "eslint-visitor-keys": "^1.0.0",
-    "typescript-estree": "5.3.0"
+    "typescript-estree": "^6.0.0"
```

Comment:
7.0.0 just came out

Reply:
@KaelWD i did pr with changes for 7.0.0 in #584